lib/config/openssh/openssh.go

```diff
@@ -52,11 +52,16 @@ Host *.{{ $clusterName }} {{ $dot.ProxyHost }}
 # Flags for all {{ $clusterName }} hosts except the proxy
 Host *.{{ $clusterName }} !{{ $dot.ProxyHost }}
     Port {{ $dot.Port }}
-    {{- if eq $dot.AppName "tsh" }}
+{{- if eq $dot.AppName "tsh" }}
     ProxyCommand "{{ $dot.ExecutablePath }}" proxy ssh --cluster={{ $clusterName }} --proxy={{ $dot.ProxyHost }}:{{ $dot.ProxyPort }} %r@%h:%p
-{{- end }}{{- if eq $dot.AppName "tbot" }}
+{{- end }}
+{{- if eq $dot.AppName "tbot" }}
+{{- if $dot.PureTBotProxyCommand }}
+    ProxyCommand "{{ $dot.ExecutablePath }}" ssh-proxy-command --destination-dir={{ $dot.DestinationDir }} --proxy-server={{ $dot.ProxyHost }}:{{ $dot.ProxyPort }} --cluster={{ $clusterName }} --tls-routing={{ $dot.TLSRouting }} --connection-upgrade={{ $dot.ConnectionUpgrade }} --resume={{ $dot.Resume }} --user=%r --host=%h --port=%p
```

Comment:
```suggestion
    ProxyCommand "{{ $dot.ExecutablePath }}" ssh-proxy-command "--destination-dir={{ $dot.DestinationDir }}" "--proxy-server={{ $dot.ProxyHost }}:{{ $dot.ProxyPort }}" "--cluster={{ $clusterName }}" --tls-routing={{ $dot.TLSRouting }} --connection-upgrade={{ $dot.ConnectionUpgrade }} --resume={{ $dot.Resume }} "--user=%r" "--host=%h" "--port=%p"
```

Let's avoid [CVE-2023-51385](https://vin01.github.io/piptagole/ssh/security/openssh/libssh/remote-code-execution/2023/12/20/openssh-proxycommand-libssh-rce.html) 😅

Reply:
Is it as safe to just put the quotes around the %h/%p/%r

Reply:
I was thinking more of "what if the destination dir has a space in it" or "what if for some reason we think that the cluster name has a space in it"; thinking about it, we should probably sanitize the values that we put in the ssh_config file (what if the destination dir has a quote character or a newline in it?).

Reply:
231161c608